priv/data/flows/webhook.json

```diff
@@ -31,6 +31,12 @@
     },
     "response_format_example": "{\"success\": true, \"media_url\": \"https://storage.googleapis.com/glific-media-bucket/Bhasini/outbound/3d92ac4f-ae77-4f85-bf8a-2a4f81e1a2b5.mp3\", \"translated_text\": \"Hello, this is a voice message generated by Glific.\"}"
   },
+  {
+    "name": "call_and_wait",
+    "body": "{\n  \"question\": \"\",\n  \"flow_id\": \"\",\n  \"endpoint\": \"\",\n  \"contact_id\": \"\",\n  \"callback_url\": \"\",\n  \"assistant_id\": \"\"\n}",
+    "headers": "{\n  \"Content-Type\": \"application/json\",\n  \"X-API-KEY\": \"<organization_api_key>\"\n}",
```

Comment:
add project-id in the body https://github.com/glific/glific/pull/4656#discussion_r2630952900